.github/workflows/ci.yml

```diff
@@ -146,3 +146,24 @@ jobs:
     - name: Create source and wheel dist
       run: |
         python -m build
+
+  server-package:
```

Comment:
Did you confirm that this CI pipeline actually fails if any of the steps return an error?

Reply:
I ran some manual negative tests. I think it would be easy to notice if it does not since we can just look in detail in the checks on Github.